template/.electron-vue/webpack.renderer.config.js

```diff
@@ -43,12 +44,21 @@ let rendererConfig = {
         }
       },
 {{/if}}
+      {
+        test: /\.scss$/,
+        use: ['vue-style-loader', 'css-loader', 'sass-loader']
+      },
+      {
+        test: /\.sass$/,
+        use: ['vue-style-loader', 'css-loader', 'sass-loader?indentedSyntax']
+      },
+      {
+        test: /\.less$/,
+        use: ['vue-style-loader', 'css-loader', 'less-loader']
+      },
```

Comment:
Maybe we could have some sort of loader generator?
Like vuejs-templates/webpack
https://github.com/vuejs-templates/webpack/blob/4ffcccb2e3b5c8f18ac5c28023ed983bb183ef96/template/build/utils.js#L33